azure-pipelines/pull-request-validation/credscan.yml

```diff
@@ -0,0 +1,25 @@
+# File: azure-pipelines\pull-request-validation\credscan.yml
+# Description: Run Credscan and PoliCheck
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger:
+  branches:
+    include:
+    - dev
+    - master
+    - release/*
+  batch: True
+
+jobs:
+- job: Phase_1
+  displayName: Phase 1
```

Comment:
Can we give these better names?

Reply:
yes ca9469fc08f2be16b6dfdbc16aec87f26f93763b